desktop_version/src/Script.cpp

```diff
@@ -2783,10 +2775,6 @@ void scriptclass::startgamemode(const enum StartMode mode)
         }
         break;
 
-#ifdef NO_CUSTOM_LEVELS
-        UNUSED(gotoerrorloadinglevel);
```

Comment:
Since the `UNUSED` macro is now unused in this file, you will need to remove `#include "Unused.h"` from the top of the file.